smoke.sh

```diff
@@ -59,6 +59,10 @@ function watchdog ()
 
     echo "Kicking in watchdog after $1 secs";
 
+    local mount_pid=$(ps auxww | grep glusterfs | grep -E "volfile-id[ =]/?$V " | awk '{print $2}' | head -1)
```

Comment:
Why not just do a `gluster volume statedump` which is the documented way to get a statedump?